src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/AdhocWorkspaceServices.cs

```diff
@@ -12,18 +12,18 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Common
 {
     public class AdhocWorkspaceServices : HostWorkspaceServices
     {
-        private static readonly Workspace DefaultWorkspace = new AdhocWorkspace();
```

Comment:
So this is was the root cause of our issues. Whenever we'd construct services in an attempt to create a workspace we'd create a second workspace here so we could re-utilize its services that were resolved via MEF